backend/api/Controllers/AreaController.cs

```diff
@@ -63,12 +63,13 @@ public async Task<ActionResult<AreaResponse>> Create([FromBody] CreateAreaQuery
                     "Succesfully created new area with id '{areaId}'",
                     newArea.Id
                 );
+
                 var response = new AreaResponse
                 {
                     Id = newArea.Id,
-                    DeckName = newArea.Deck.Name,
-                    PlantCode = newArea.Plant.PlantCode,
-                    InstallationCode = newArea.Installation.InstallationCode,
+                    DeckName = newArea.Deck?.Name ?? string.Empty,
+                    PlantCode = newArea.Plant?.PlantCode ?? string.Empty,
```

Comment:
#967 Address making the plant and installation required and #970 address the warnings and this fix. Is it better to have the fix here as well or address it through the other two?